### ISSUE TITLE
Transfering items from shopping cart for just logged in customer is not working

### ISSUE THREAD

**Issue by dzarda86** [shopsys/shopsys#588]
Unregistered user's items of shopping cart are not merged into new shopping cart of justlogged in customer.

### What is happening
I am using Shopsys 7.0.0.beta1

1) As an unregistered user I add few items into my shopping cart
2) I log in into my account (lets say just registered)
3) Items from my shopping cart are gone

--
4) After logging out they are back again

--

Hint: CartMigrationFacade ->mergeCurrentCartWithCart is triggered only after logout. It is not triggered after login (i am using LoginService->loginUser() method)
